src/main/java/hudson/plugins/spotinst/cloud/monitor/AwsSpotinstCloudInstanceTypeMonitor.java

```diff
@@ -0,0 +1,79 @@
+package hudson.plugins.spotinst.cloud.monitor;
+
+import hudson.Extension;
+import hudson.model.AdministrativeMonitor;
+import hudson.plugins.spotinst.cloud.AwsSpotinstCloud;
+import hudson.plugins.spotinst.common.SpotinstContext;
+import jenkins.model.Jenkins;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Extension
+public class AwsSpotinstCloudInstanceTypeMonitor extends AdministrativeMonitor {
+    //region members
+    private static final Logger LOGGER = LoggerFactory.getLogger(AwsSpotinstCloud.class);
+    List<String> invalidInstances;
+    //endregion
+
+    //region Overrides
+    @Override
+    public boolean isActivated() {
+        boolean     retVal                = false;
+        Set<String> allCloudsInvalidTypes = findInvalidInstancesFromClouds();
+
+        SpotinstContext.getInstance().setInvalidInstanceTypes(new ArrayList<>(allCloudsInvalidTypes));
```

Comment:
just return true - try it

Reply:
Put always an empty alert. should do the check